src/Cli/ConfigGenerator.cs

```diff
@@ -138,49 +143,48 @@ public static bool TryCreateRuntimeConfig(InitOptions options, out string runtim
                 return false;
             }
 
-            RuntimeConfig runtimeConfig = new(
+            string dabSchemaLink = loader.GetPublishedDraftSchemaLink();
+
+            runtimeConfig = new(
                 Schema: dabSchemaLink,
                 DataSource: dataSource,
-                RuntimeSettings: GetDefaultGlobalSettings(
-                    options.HostMode,
-                    options.CorsOrigin,
-                    options.AuthenticationProvider,
-                    options.Audience,
-                    options.Issuer,
-                    restPath,
-                    !options.RestDisabled,
-                    options.GraphQLPath,
-                    !options.GraphQLDisabled),
-                Entities: new Dictionary<string, Entity>());
-
-            runtimeConfigJson = JsonSerializer.Serialize(runtimeConfig, GetSerializationOptions());
+                Runtime: new(
+                    Rest: new(!options.RestDisabled, restPath ?? RestRuntimeOptions.DEFAULT_PATH),
```

Comment:
Are you worried that the state of the JSON file might be misleading to someone who reads it in this case? If it's disabled but there's a set path, it will be ignored anyway, since it's not relevant.

Added code to forcably disable REST for CosmosDB.